gen/src/generate/idna/shared/version.rs

```diff
@@ -0,0 +1,55 @@
+// TODO: Unify this with generate::ucd::shared::version
```

Comment:
Oh, this is a `shared` module under `unic-gen`, not `unic-ucd`. I thought it's about `src/tables/` dirs under `/unic/` components.

Sure, whatever makes it simpler.

Also, it reminded me that we *may* need API to expose the version of other data sources, as well, like IDNA version, Emoji version, etc. I'm going to file a separate issue about the implementation, but something to keep in your mind here.